test/azure/terraform_azure_sqlmanagedinstance_example_test.go

```diff
@@ -0,0 +1,67 @@
+//go:build !azure
+// +build !azure
```

Comment:
I think should be used same build parameters as for other `azure` tests

```
//go:build azure
// +build azure
```

Now, the regular CICD test job tries to run TestTerraformAzureSQLManagedInstanceExample and fails since `az` is not configured, other Azure tests aren't attempted to be executed

```
TestTerraformAzureSQLManagedInstanceExample 2023-06-25T21:29:15Z logger.go:66: ╵
TestTerraformAzureSQLManagedInstanceExample 2023-06-25T21:29:15Z retry.go:99: Returning due to fatal error: FatalError{Underlying: error while running command: exit status 1; ╷
│ Error: building AzureRM Client: please ensure you have installed Azure CLI version 2.0.79 or newer. Error parsing json result from the Azure CLI: launching Azure CLI: exec: "az": executable file not found in $PATH.
│ 
│   with provider["registry.terraform.io/hashicorp/azurerm"],
│   on main.tf line 12, in provider "azurerm":
│   12: provider "azurerm" {
│ 
╵}
    apply.go:15: 
        	Error Trace:	/home/circleci/project/test/azure/apply.go:15
        	            				/home/circleci/project/test/azure/terraform_azure_sqlmanagedinstance_example_test.go:46
        	Error:      	Received unexpected error:
        	            	FatalError{Underlying: error while running command: exit status 1; ╷
        	            	│ Error: building AzureRM Client: please ensure you have installed Azure CLI version 2.0.79 or newer. Error parsing json result from the Azure CLI: launching Azure CLI: exec: "az": executable file not found in $PATH.
        	            	│ 
        	            	│   with provider["registry.terraform.io/hashicorp/azurerm"],
        	            	│   on main.tf line 12, in provider "azurerm":
        	            	│   12: provider "azurerm" {
        	            	│ 
        	            	╵}
        	Test:       	TestTerraformAzureSQLManagedInstanceExample
```
[circle-test.txt](https://github.com/gruntwork-io/terratest/files/11866869/circle-test.txt)


References:

https://github.com/gruntwork-io/terratest/blob/master/test/azure/terraform_azure_aci_example_test.go#L2

Reply:
Hi, correct

Reply:
Hi @denis256, looking  at the circleci yaml file, I see that the azure tests are not compiled and not executed. I am not sure which part of the workflow triggered the execution of the azure tests. Can you please point me to the test stage/task where is got triggered. Also can you share how/where a subset of the tests are mared as short? I see in the log file you shared earlier that the test cmd  is using the -short flag ' go test -v -timeout 45m -parallel 128  -p 1 ./... -short'

Reply:
Step:
```
 - run: run-go-tests --packages "-p 1 ./..." | tee /tmp/logs/test_output.log
```

Included execution of `TestTerraformAzureSQLManagedInstanceExample`

![image](https://github.com/gruntwork-io/terratest/assets/10694338/2daf0ed3-b833-4f18-a35b-5328b4709f42)

I suspect it is because of(I need to do more tests to validate):
```
//go:build !azure
// +build !azure
```

Other tests have:
```
//go:build azure
// +build azure
```

Reply:
In https://github.com/gruntwork-io/terratest/tree/add-sqlmi-module-azure-test
I set
```
//go:build azure
// +build azure
```

and `TestTerraformAzureSQLManagedInstanceExample` wasn't executed

Reply:
@denis256 I updated the PR to skip the test for the short executions, and update the tags as well. Testing locally the test has not been picked up by the go test cmd.  please take a look when you get a chance and share the Circle ci log file as well.